src/visualizer/statisticsMath.js

```diff
@@ -73,4 +75,24 @@
     function getLSValuesOfGroup(group, learnStyleName){
         return group.students.map((s) => s.criteria.learningStyles[learnStyleName]);
     }
+
+    /**
+     * @summary Creates and returns a new array with the subject scores
+     * of the subject with the subject name provided in the subjectName argument
+     * @param {object} group the group from which to get the data
+     * @param {string} subjectName the name of the subject
+     * @returns {object} returns an object with the students of the groups values in the given subjects, the key is the students idx in the groups.students object
```

Comment:
Det er et number array der bliver returned

Reply:
@returns {object} ->@returns {number[]}

Reply:
Fix teksten også "returns an array..."

Reply:
Done, men du overså:
    /**
     * @summary Creates and returns a new array with the learningstyle values(-11 to 11)
     * of the learningstyle with the learningstyle(e.g. active, visual, sensing..) name provided in the learnStyleName argument
     * @param {object} group the group from which to get the data
     * @param {string} learnStyleName the name of the learningstyle e.g. "activeReflective"
     * @returns {number[]} returns an array with the students of the groups values in the given learningstyle, the key is the students idx in the groups.students object
     */
    function getLSValuesOfGroup(group, learnStyleName){
        return group.students.map((s) => s.criteria.learningStyles[learnStyleName]);
    }